client/dashboard/sites/logs/index.tsx

```diff
@@ -90,19 +93,20 @@ function SiteLogs( { logType }: { logType: LogType } ) {
 			router.navigate( { to: `/sites/${ siteSlug }/logs/server` } );
 		}
 	};
-
+	const hasActivityLogAccess =
+		hasHostingFeature( site, HostingFeatures.ACTIVITY_LOG ) ||
+		hasPlanFeature( site, HostingFeatures.ACTIVITY_LOG );
+	// hide the datepicker if the user doesn't have access to activity logs or doesn't have logging feature at all
+	const shouldShowDateRangePicker =
```

Comment:
CMIIW but I think this is simply

```suggestion
	const shouldShowDateRangePicker = hasHostingFeature( site, HostingFeatures.LOGS );
```

Reply:
I think you're correct here, although I wonder if there are some edge cases I might miss. After some testing, though, it seems to me that if we don't have logs, at least on WordPress.com, then we won't also have Activity Logs.